examples/report_update_request_response_example.txt

```diff
@@ -0,0 +1,24 @@
+REQUEST:
```

Comment:
Should we put request and response in different files, without comments, verbatim?